compression/lpf.py

```diff
@@ -0,0 +1,73 @@
+
+from string_indexing import lcp, suffix_array
+
+
+def naive(w, n):
```

Comment:
I think that the whole file best belongs to string_indexing, as it's often developed in relation to SA, LCP and suffix trees. Please "git mv" this file to that directory.

Reply:
Ack